dbinit/src/main/liquibase/updates/changelog-0008-yd-326.yml

```diff
@@ -0,0 +1,64 @@
+databaseChangeLog:
+- changeSet:
+    id: 1508867617816-1
+    author: avdijk (generated)
+    changes:
+    - createTable:
+        columns:
+        - column:
+            constraints:
+              nullable: false
+            name: id
+            type: VARCHAR(255)
+        - column:
+            name: png_bytes
+            type: TINYBLOB
```

Comment:
See comment on ``UserPhoto.pngBytes``. A TINYBLOB is too small to hold a photo.

Reply:
Will change to MEDIUMBLOB